src/libs/registry/docset.h

```diff
@@ -76,7 +79,7 @@ class Docset
         ZDash
     };
 
-    QSqlDatabase database() const;
+    Util::SQLiteDatabase *m_db;
```

Comment:
Please move it somewhere below, after `m_symbols` would be nice. Also ` = nullptr`.